package.json

```diff
@@ -78,7 +78,7 @@
     "karma-jasmine": "~1.1.1",
     "karma-jasmine-html-reporter": "^0.2.2",
     "node": "^8.16.0",
-    "protractor": "^6.0.0",
+    "protractor": "~5.3.0",
```

Comment:
Dropping this below 6 introduces 2 new high level security warnings from NPM.

Reply:
Oh dang :( yea it wasn't working for me on 6. I will try again tho

Reply:
I updated to 5.4.3 which seems to work. Also admin is on 5.3 so we could update that. I think 5.4.3 is the latest version based on their npm [site](https://www.npmjs.com/package/protractor)

Reply:
Nice, this removes the high vulnerabilities.